### PR TITLE
Fix for #5495 : add setTrackbarMin

### PR DESCRIPTION
**Pull request by aman11dhanpat**

### CODE REVIEW

**Comment by aman11dhanpat** [comment]
Is it possible to get list of errors from pullrequest build server?

**Comment by alalek** [comment]
@aman11dhanpat 
1) Click "Show all checks" near "All checks have failed"
2) Click "Details" link which will go [here](http://pullrequest.opencv.org/#/summary/opencv)

**Comment by StevenPuttemans** [comment]
Thank you! Please add a description to your PR :) It can help for future searches.

**Comment by vpisarev** [comment]
:+1:

File: modules/highgui/src/window_QT.cpp [hunk 1]
```diff
@@ -664,12 +664,29 @@ CV_IMPL void cvSetTrackbarMax(const char* name_bar, const char* window_name, int
...
 
 
+CV_IMPL void cvSetTrackbarMin(const char* name_bar, const char* window_name, int minval)
+{
+    if (minval >= 0)
+    {
+        QPointer<CvTrackbar> t = icvFindTrackBarByName(name_bar, window_name);
+        if (t)
+        {
+            int maxval = t->slider->getMaximum();
```
Comment (paroj) [inline modules/highgui/src/window_QT.cpp]: have you compiled this? there is no such call for QSlider

**Comment by aman11dhanpat** [comment]
I forgot to compile with qt. Thought gtk was enough. 
I'll raise a pr soon.

**Comment by paroj** [comment]
do not bother #5570

**Pull request merged**